src/handlers/mod.rs

```diff
@@ -183,7 +192,27 @@ impl InputMethodHandler for State {
     }
 }
 
+impl KeyboardShortcutsInhibitHandler for State {
+    fn keyboard_shortcuts_inhibit_state(&mut self) -> &mut KeyboardShortcutsInhibitState {
+        &mut self.niri.keyboard_shortcuts_inhibit_state
+    }
+
+    fn new_inhibitor(&mut self, inhibitor: KeyboardShortcutsInhibitor) {
+        inhibitor.activate();
```

Comment:
```suggestion
        // FIXME: show a confirmation dialog with a "remember for this application" kind of toggle.
        inhibitor.activate();
```

Reply:
I'm not quite sure how such a dialog would work in niri (how do we remember it? just based off `app-id`? what about when it's done on a layer shell surface? also, how do we display the dialog? with pangocairo or though gnome portal somehow?), or if it even makes sense to have (this isn't exactly a "permission" a client asks for as on iOS/Android; if you have a client that abuses it, that client should be fixed really), but i've added the comment nonetheless because these questions don't need to be resolved in this PR if the implementation of such a feature is done separately